packages/core/src/combinator/mergeConcurrently.js

```diff
@@ -78,11 +77,16 @@ class Outer {
     this.active = false
     this.pending.length = 0
     this.disposable.dispose()
-    this.current.dispose()
+    disposeAll(this.current).dispose()
+    // this.current.dispose()
```

Comment:
Done in c2b6b9a

Reply:
ooh i thought those comments were like what it was actually doing like before it was evident in the code but then you switched to an array it was as clear i guess... so i thought the comments were there because you want to keep the clarity welp next i will ask instead of assuming